frappe_graphql/frappe_graphql/types/role.graphql

```diff
@@ -0,0 +1,59 @@
+type Role implements BaseDocType {
+  doctype: String
+  name: String
+  owner: User!
+  creation: String
+  modified: String
+  modified_by: User!
+  parent: String
+  parentfield: String
+  parenttype: String
+  idx: Int
+  docstatus: Int
+  owner__name: String!
+  modified_by__name: String!
+  role_name: String!
+  home_page: String
+  # restrict_to_domain: Domain
```

Comment:
why comment this ? add the type if its causing issues right ? @assemmarwan @faztp12